include/emds/emds.h

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/** @file
+ *  @defgroup emds Emergency Data Storage
+ *  @ingroup emds
+ *  @{
+ *  @brief Emergency Data Storage API
+ */
+
+#ifndef EMDS_H__
+#define EMDS_H__
+
+#include <stddef.h>
+#include <sys/types.h>
+#include <sys/util.h>
+#include <sys/slist.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @struct emds_entry
+ *
+ * Information about entries used to store data in the emergency data storage.
+ */
+struct emds_entry {
```

Comment:
Should be any variable to keep the information it is static or dynamic entry?

Reply:
There is places that it does not fit to have this dynamic, like the RPL list. That does not have any init function that can add this, so for RPL list we need the static way of adding the data pointer.